templates/master/00-master/two-node-with-fencing/files/podman-etcd.yaml

```diff
@@ -0,0 +1,5 @@
+mode: 0755
```

Comment:
minor change to this file. can we set
```
overwrite: true
```
I had to set an empty file in the target location to pass the kola tests in RHCOS